citar-cache.el

```diff
@@ -209,7 +211,7 @@ re-hashing the file contents."
                    (buffer-hash)))))
     `(:size ,size :mtime ,mtime :hash ,hash :fields ,fields)))
 
-(defun citar-cache--update-bibliography-p (oldprops newprops)
+(defun citar-cache--update-default-bibliography-p (oldprops newprops)
```

Comment:
One question on this; what's the implications of the very last line of this docstring for this PR?